ain_benchmark.js

```diff
@@ -243,7 +256,8 @@ function printTestResult(testList) {
     }
     console.log();
   }
-  console.log(`Total TPS: ${totalTps.toFixed(5)}`);
+  console.log(`Total TPS (X): ${totalTps.toFixed(5)}`);
+  console.log(`Number of paths in state database (Y): ${numberOfPaths}`);
```

Comment:
What about more direct message? 'Number of shards (sharding paths) (Y): ...'

Reply:
Done! Thanks